README.md

```
@@ -15,24 +15,31 @@ GitHub 프로젝트의 의존성 취약점을 3D 젠가 형태로 시각화하
+| Prisma | 7 | ORM |
 | PostgreSQL | 17 | 데이터베이스 |
 | Tailwind CSS | 4 | 스타일링 |
+| shadcn/ui | - | UI 컴포넌트 |
+| TanStack Query | 5 | 서버 상태 관리 |
+| Vitest | 4 | 테스트 |
+| Storybook | 10 | 컴포넌트 문서화 |
+| Playwright | 1 | E2E 테스트 |
 
 ## 📂 프로젝트 구조
 
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)